README.md

```diff
@@ -163,3 +192,5 @@ Security
 
 Security issues in K3s can be reported by sending an email to [security@k3s.io](mailto:security@k3s.io).
 Please do not file issues about security issues.
+
+[GitHub Issues]: https://github.com/k3s-io/k3s/issues
```

Comment:
I don't think this belongs under Security since we don't want people creating issues for security stuff.

Reply:
that's a short cut for string "GitHub Issues" take a look line 110.
It will not be rendered, but it can be reused in case we want to add same link on different places on the readme.